google/resource_storage_bucket.go

```diff
@@ -351,6 +382,11 @@ func resourceStorageBucketRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("storage_class", res.StorageClass)
 	d.Set("location", res.Location)
 	d.Set("cors", flattenCors(res.Cors))
+	versioning := make([]map[string]interface{}, 0, 1)
+	versioning = append(versioning, map[string]interface{}{
```

Comment:
extract this in a new method called `flattenBucketVersioning` for consistency.

See this method for an example: https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_sql_database_instance.go#L1037